_maps/shuttles/shiptest/ntsv_osprey.dmm

```diff
@@ -6617,6 +6617,22 @@
 	},
 /turf/open/floor/plasteel,
 /area/ship/hallway/aft)
+"Vi" = (
+/obj/structure/closet/secure_closet{
+	icon_state = "hop";
+	name = "\proper first officer's locker";
+	req_access_txt = "57"
+	},
+/obj/item/storage/backpack,
+/obj/item/clothing/under/rank/command/lieutenant/blue,
+/obj/item/clothing/gloves/color/white,
+/obj/item/clothing/shoes/laceup,
+/obj/item/clothing/suit/armor/vest/hop,
+/obj/item/clothing/head/collectable/HoP,
+/obj/item/clothing/glasses/sunglasses,
+/obj/item/assembly/flash/handheld,
+/obj/item/storage/box/PDAs,
+/obj/item/storage/box/PDAs,
```

Comment:
```suggestion
/obj/item/storage/box/PDAs)
```
Try not to webedit map files because you'll end up breaking the map file like this